.vscode/launch.json

```
@@ -18,11 +26,13 @@
       "preLaunchTask": "Build Extension"
     },
     {
+      // Used for testing theextension with a local build of the LSP server (in `target/debug`).
```

```suggestion
      // Used for testing the extension with a local build of the LSP server (in `target/debug`).
```

Oof, I messed this up.